RogueEssence/Menu/MenuManager.cs

```diff
@@ -170,6 +170,16 @@ public IEnumerator<YieldInstruction> SetWaitMenu(bool anyInput)
             WaitMenu box = new WaitMenu(anyInput);
             yield return CoroutineManager.Instance.StartCoroutine(ProcessMenuCoroutine(box));
         }
+        public IEnumerator<YieldInstruction> SetWaitMenu(List<FrameInput.InputType> inputs)
```

Comment:
I'm trying to keep the number of open menu calls in this class low.  Since this window is rather straightforward, remove these two convenience functions and simply create the windows in the calling code, to be passed into ProcessMenuCoroutine.